class/KibblesTasty; Artificer (Revised).json

```diff
@@ -1,7569 +1,7831 @@
-			"subclassSource": "ArtificerRevised",
-			"level": 1,
-			"header": 1
-		},
-		{
-			"name": "Artificial Strength",
-			"entries": [
-				"When you don your Warplate Gauntlet or as an action while wearing it, you can dedicate some of your intelligence to fully controlling the power of the gauntlet. You can reduce your current and maximum Intelligence score to increase your current Strength ability score by the same amount, but you can only raise your Strength ability score up what your Intelligence ability score was before engaging Artificial Strength. You can stop using Artificial Strength at any time, and it automatically ends if your gauntlet is removed."
-			],
-			"source": "ArtificerRevised",
-			"className": "Artificer (Revised)",
-			"classSource": "ArtificerRevised",
-			"subclassShortName": "Warsmith",
-			"subclassSource": "ArtificerRevised",
-			"level": 1,
-			"header": 1
-		},
-		{
-			"name": "Warsmith's Armor",
-			"source": "ArtificerRevised",
-			"className": "Artificer (Revised)",
-			"classSource": "ArtificerRevised",
-			"subclassShortName": "Warsmith",
-			"subclassSource": "ArtificerRevised",
-			"level": 3,
-			"header": 2,
-			"entries": [
-				"At 3rd level, you've attained the knowledge of forging and arcane tinkering sufficient to create a set of armor that augments and expands your abilities from a standard, nonmagical, set of heavy armor using resources you've gathered. This process takes 8 hours to complete, as well as a place to forge and incorporates a Warplate Gauntlet (they do not require separate attunement).",
-				"While wearing your armor, your Strength ability score increases by 2, and your maximum Strength ability score becomes 22.",
-				"You can create a new set of armor by forging it from a set of gathered and purchased materials in a process that takes 2000 gold pieces and eight hours.",
-				"You can create multiple sets of armor, but you can only be attuned to one of them at a given time, and you can only change which one you are attuned to during a long rest. If you create a new set of Warplate, you can apply a number of Upgrades equal to the value on the class table, applying each at the level you get it on the class table.",
-				"When you create your armor, you can create a heavy plated Warplate, a magically enhanced set of gear, Warsuit, or you can integrate your changes directly into your body as an Integrated Armor.",
-				{
-					"type": "table",
-					"colLabels": [
-						"Armor Name",
-						"AC",
-						"Weight",
-						"Strength Requirement",
-						"Stealth",
-						"Properties"
-					],
-					"colStyles": [
-						"col-1 text-center",
-						"col-3 text-center",
-						"col-1 text-center",
-						"col-1 text-center",
-						"col-1 text-center",
-						"col-5 text-center"
-					],
-					"rows": [
-						[
-							"Warplate",
-							"18",
-							"75 lbs.",
-							"-",
-							"Disadvantage",
-							"Heavy Armor, Powerful Build*"
-						],
-						[
-							"Warsuit",
-		
```

Comment:
```suggestion
```

Reply:
🤔 either I've messed something up, or GitHub is failing to deal with this massive file. 
Either way, you've got a nameless, text-less feature in there that you need to remove.

Reply:
If you're trying to give suggestions on a huge file then github gets lines wrong yeah. :(